web/src/components/Settings/MemberSection.tsx

```diff
@@ -20,17 +22,31 @@ const PreferencesSection = () => {
     createUserUsername: "",
     createUserPassword: "",
   });
+  const [userNameQueryText, setUserNameQueryText] = useState("");
   const [userList, setUserList] = useState<User[]>([]);
+  const [showUserList, setShowUserList] = useState<User[]>([]);
```

Comment:
We can get `showUserList` from `userList` with `queryText`. e.g.

<img width="789" alt="image" src="https://user-images.githubusercontent.com/24653555/204268081-b7b7f9ba-c007-4111-b4f2-b99e24f44f24.png">